docker/nautilus.sh

```diff
@@ -0,0 +1,257 @@
+#make working directory
+[[ -d "$WORKING_DIR" ]] || mkdir "$WORKING_DIR"
+#create variable for tezos-network(alphanet or mainnet)
+tezosnetwork=`cat "$PATH_TO_CONFIG"/tezos/tezos_network.txt`
+
+docker network create nautilus
+
+build_conseil () { 
+	docker container stop conseil-"$DEPLOYMENT_ENV"
+	docker container rm conseil-"$DEPLOYMENT_ENV"
+
+	CONSEIL_WORK_DIR="$WORKING_DIR"/conseil-"$DEPLOYMENT_ENV"
+    mkdir "$CONSEIL_WORK_DIR"
+    cd "$CONSEIL_WORK_DIR"
+
+    . "$DIR"/app/conseil/build.sh
+
+    cp "$PATH_TO_CONFIG"/conseil/conseil.conf ./conseil.conf
+    conseil_conf_file=./conseil.conf
+    {
+    read line1
+    read line2
+    read line3
+    } < "$PATH_TO_CONFIG"/conseil/credentials.txt
+    line1=`echo $line1`
+    line2=`echo $line2`
+    line3=`echo $line3`
+
+    sed -i "s/databaseName=.*/$line1/g" "$conseil_conf_file"
+    sed -i "s/user=.*/$line2/g" "$conseil_conf_file"
+    sed -i "s/password=.*/$line3/g" "$conseil_conf_file"
+
+    cp "$conseil_conf_file" ./build/
+    cp "$PATH_TO_CONFIG"/conseil/runconseil-lorre.sh ./build/
+
+
+
+    cp ./Conseil/src/main/resources/logback.xml ./build/
+    #cp "$PATH_TO_CONFIG"/conseil/runconseil-lorre.sh ./build/
+
+
+    docker build -f "$DIR"/app/conseil/dockerfile -t conseil-"$DEPLOYMENT_ENV" .
+    rm ./build
+   	docker run --name=conseil-"$DEPLOYMENT_ENV" --network=nautilus -d -p 1337:1337 conseil-"$DEPLOYMENT_ENV"
+
+	yes | docker system prune
+}
+
+build_postgres () {
+    #declaring persistent volumes
+    #current volume created check
+    [[ -d $HOME/volumes ]] || mkdir $HOME/volumes
+    [[ -d $HOME/volumes/pgdata-"$DEPLOYMENT_ENV" ]] || mkdir $HOME/volumes/pgdata-"$DEPLOYMENT_ENV"
+
+    docker volume create --driver local --opt type=none --opt o=bind --opt device=$HOME/volumes/pgdata-"$DEPLOYMENT_ENV" pgdata-"$DEPLOYMENT_ENV"
+
+    docker container stop postgres-"$DEPLOYMENT_ENV"
+	docker container rm postgres-"$DEPLOYMENT_ENV"
+
+    #check out schema and put it in the right place
+
+
+    POSTGRES_WORK_DIR="$WORKING_DIR"/postgres-"$DEPLOYMENT_ENV"
+    mkdir "$POSTGRES_WORK_DIR"
+
+    cp "$DIR"/app/postgres/dockerfile "$POSTGRES_WORK_DIR"/dockerfile
+    postgres_dockerfile="$POSTGRES_WORK_DIR"/dockerfile
+    #change postgres databasename, username, and password
+    {
+    read line1
+    read line2
+    read line3
+    } < "$PATH_TO_CONFIG"/postgres/credentials.txt
+    line1=`echo $line1`
+    line2=`echo $line2`
+    line3=`echo $line3`
+
+    sed -i "s/ENV POSTGRES_USER=.*/$line1/g" "$postgres_dockerfile"
+    sed -i "s/ENV POSTGRES_PASSWORD=.*/$line2/g" "$postgres_dockerfile"
+    sed -i "s/ENV POSTGRES_DB=.*/$line3/g" "$postgres_dockerfile"
+
+
+
+    #check out schema and put it in the right place
+    mv "$PATH_TO_CONFIG"/postgres/conseil.sql "$PATH_TO_CONFIG"/postgres/conseil.sql.bak
+    wget https://raw.githubusercontent.com/Cryptonomic/Conseil/master/doc/conseil.sql > "$PATH_TO_CONFIG"/postgres/conseil.sql
```

Comment:
This file should be downloaded into the working directory and not the configuration directory. The former is transient whereas the latter is under source control.

Reply:
fixed.